doc/source/api/core/solver/fielddata.rst

```diff
@@ -149,26 +149,22 @@ The ``add_get_<items>_request`` methods combine requests for multiple fields in
 - ``add_get_surfaces_request`` adds a surfaces request.
 - ``add_get_scalar_fields_request`` adds a scalar fields request.
```

Comment:
@PipKat  Kindly review the changes in this file. Thanks

Reply:
@ajain-work Sorry but I was out of commission for several days due to Hurricane Ian here in Florida. I'll take a look at your Vale errors.

Reply:
@PipKat Thanks a lot for reviewing and providing feedback.